girder/cumulus/plugin_tests/volume_test.py

```diff
@@ -473,11 +482,13 @@ def test_attach_volume(self, get_ec2_client):
             volume_id, self._trad_cluster_id)
         r = self.request(url, method='PUT',
                          type='application/json', body=json.dumps(body),
-                         user=self._cumulus)
+                         user=self._user)
         self.assertStatus(r, 400)
```

Comment:
I was getting an error which I traced back to a query to mongo coming back with None. Digging around in mongo I noticed that the aws profiles it was trying to retrieve have the userId associated with the 'user' login rather than the 'cumulus' login. My changes make this consistent with  aws_test.py where all the request are passed _user rather than _cumulus.